Deck.cs

```diff
@@ -14,9 +14,33 @@ public bool IsEmpty()
             }
             return false;
         }
+        public void SortCards(List<Card> selectedCards, CardsAttributes attribute)
+        {
+            selectedCards.OrderBy(card => card[attribute]);
+        }
+        public bool IsTie(List<Card> selectedCards, CardsAttributes attribute)
+        {
+            var highestCard = selectedCards.OrderBy(card => card[attribute]).First();
+            
+            for (int i=1; i<selectedCards.Count; i++)
+            {
+                if(selectedCards[i][attribute] == highestCard[attribute])
+                {
+                    return true;
+                }
+            }
+            return false;
```

Comment:
you don't need to compare all cards only the second one with the first when they are sorted, the statement can be much simplier